app/views/content/train-to-be-a-teacher/teacher-training-interview.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
@EllieNodder @philspray 

Is this the best URL for this page based on search terms etc (it might be - I've no idea!)?

Reply:
I think it's the best option in terms of the search volumes.